transcode/transcode_jobs.go

```diff
@@ -41,8 +40,6 @@ func (t *ParallelTranscoding) Start() {
 	t.completed.Add(config.TranscodingParallelJobs)
 	for index := 0; index < config.TranscodingParallelJobs; index++ {
 		go t.workerRoutine()
-		// Add some desync interval to avoid load spikes on segment-encode-end
-		time.Sleep(config.TranscodingParallelSleep)
```

Comment:
Unrelated but I spotted this - I think this was us being concerned about a problem that doesn't exist and is safe to remove